src/main/java/com/example/pet_shelter/controller/PersonRESTController.java

```diff
@@ -0,0 +1,52 @@
+package com.example.pet_shelter.controller;
+
+
+import com.example.pet_shelter.entity.Person;
+import com.example.pet_shelter.service.PersonService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class PersonRESTController {
```

Comment:
you already put annotation @RestController no need to declare it in the name  PersonRESTController (REST)